qiskit/visualization/circuit_visualization.py

```diff
@@ -137,7 +137,7 @@ def circuit_drawer(circuit,
             when the ``output`` kwarg is set to use the ``mpl`` backend. It
             will be silently ignored with all other outputs.
         initial_state (bool): Optional. Adds ``|0>`` in the beginning of the wire.
-            Only used by the ``text``, ``latex`` and ``latex_source`` outputs.
+            Used by the ``text``, 'mpl', ``latex`` and ``latex_source`` outputs.
```

Comment:
If it's for all of them, we can remove that sentence (as in other for-all args)
```suggestion
```

Reply:
Had the same thought.